### PR TITLE
[Feature-6423][TIS] Rename tis to pigeon

### PR DESCRIPTION
**Pull request by jixiang155**
[#6423](https://github.com/apache/dolphinscheduler/issues/6423)

### CODE REVIEW

File: dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js [hunk 1]
```diff
@@ -586,7 +586,7 @@ export default {
   TargetDataBase: '目标库',
   TargetTable: '目标表',
   TargetJobName: 'TIS目标任务名',
-  'Please enter TIS DataX job name': '请输入TIS DataX任务名',
+  'Please enter PIGEON DataX job name': '请输入TIS DataX任务名',
```
Comment (zhongjiajie) [inline dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js]: Maybe we should change the describe here, about the keyword `DataX` and `TIS`, BTW, please change `en_US.js` too

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/pigeon.vue [hunk 2]
```diff
@@ -23,7 +23,7 @@
           type="input"
           size="small"
           v-model="targetJobName"
-          :placeholder="$t('Please enter TIS DataX job name')">
+          :placeholder="$t('Please enter PIGEON DataX job name')">
```
Comment (zhongjiajie) [inline dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/pigeon.vue]: Maybe remove the DATAx keyword here too

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/canvas/taskbar.scss [hunk 3]
```diff
@@ -174,8 +174,8 @@
             &.icos-switch {
               background-image: url("../images/task-icos/switch_hover.png");
             }
-            &.icos-tis {
-              background-image: url("../images/task-icos/tis_hover.png");
+            &.icos-pigeon {
```
Comment (zhongjiajie) [inline dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/canvas/taskbar.scss]: Why we have two `tis` icos here in L159 and L177

Comment (baisui1981) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/canvas/taskbar.scss]: add duplicate, should be reserved one

File: dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js (hunk 1 above)
Comment (baisui1981) [reply dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js]: > 'Please enter DataX job name': '请输入DataX任务名',

this shoud be OK

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/canvas/taskbar.scss (hunk 3 above)
Comment (jixiang155) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/canvas/taskbar.scss]: has done

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/pigeon.vue (hunk 2 above)
Comment (jixiang155) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/pigeon.vue]: > Maybe remove the DATAx keyword here too

has removed

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/canvas/taskbar.scss (hunk 3 above)
Comment (jixiang155) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/canvas/taskbar.scss]: > add duplicate, should be reserved one

done

File: dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js (hunk 1 above)
Comment (jixiang155) [reply dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js]: > Maybe we should change the describe here, about the keyword `DataX` and `TIS`, BTW, please change `en_US.js` too

description has been improved, and en_US.js lack these description,  now, has added

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Comment by zhongjiajie** [comment]
Could we add describe about what pigeon is and how could it do?  @jixiang155 @lenboo

**Comment by baisui1981** [comment]
> Could we add describe about what pigeon is and how could it do? @lenboo

I have through about it ，but looked around the ds site，have not found a good place to add the introducing about the pigeon，do you have some suggestion？

**Comment by zhongjiajie** [comment]
Is there a good choose to add to https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/system-manual.html section `7. Task node type and parameter settings`

**Review by CalvinKirs — CHANGES_REQUESTED** [review]
hi, and the back-end-related classes have not been modified.

**Review by CalvinKirs — APPROVED** [review]
Thank you for your contribution, I will merge the front-end first, and the back-end can resubmit the PR later

**Pull request merged**